File: src/main/java/br/com/tech/challenge/cpf/ResultCPFValidator.java
package br.com.tech.challenge.cpf;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class ResultCPFValidator {

    private static final String ABLE_TO_VOTE = "ABLE_TO_VOTE";
    private String status;
    public static ResultCPFValidator ableToVote() {
        return new ResultCPFValidator(ABLE_TO_VOTE);
    }
    public Boolean isValid() {
        return ABLE_TO_VOTE.equals(status);
    }
}

File: src/main/java/br/com/tech/challenge/cpf/ResultCPFValidator.kt
package br.com.ntconsult.challenge.proxy.cpf

private const val ABLE_TO_VOTE = "ABLE_TO_VOTE"

class ResultCPFValidator(val status: String?) {

    fun isValid() = status == ABLE_TO_VOTE

    companion object {
        fun default() = ResultCPFValidator(ABLE_TO_VOTE)
    }
}